sklearn/svm/tests/test_svm.py

```diff
@@ -575,6 +575,24 @@ def test_linearsvc_parameters():
                          svm.LinearSVC(loss="l3").fit, X, y)
 
 
+def test_intercept_scaling_warning():
+    clf = svm.LinearSVC(fit_intercept=True)
+    assert_warns_message(UserWarning,
+                         "liblinear does not regularize the intercept. "
```

Comment:
Liblinear *does* regularise the intercept. That's the problem! You've inverted the statement, or I did.

Reply:
I wrote this based on your comment _This is to improve awareness of liblinear's failure to regularise the intercept._ And you didn't complain about this part of the text on the first review :D 

Shall I change it to _liblinear regularizes the intercept. ...._?

Reply:
Yes, please. I assumed it was my error. Negations aren't my forté! :D

Reply:
Done.